### ISSUE TITLE
CORS Issue

### ISSUE THREAD

**Issue by AlaaMX** [Crypto-APIs/Node.js-Library#17]
Access to fetch at 'https://api.cryptoapis.io/v1/bc/btc/testnet/address/mgA6AjT1ju2B7er3D1qEBrhmVrNeNgTnSv' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

Getting it even with providing key

**Comment by Zergity** [comment]
@AlaaMX I have the same problem, did you solve it?

**Comment by AlaaMX** [comment]
@Zergity do not fetch from client , do it from the server side !

**Comment by Zergity** [comment]
> @Zergity do not fetch from client , do it from the server side !

Right, cryptoapis blocks the client request, which is against my purpose, so I have to switch to other API. Thanks.

**Comment by AlaaMX** [comment]
> > @Zergity do not fetch from client , do it from the server side !
> 
> Right, cryptoapis blocks the client request, which is against my purpose, so I have to switch to other API. Thanks.

Yes, They are doing That for Security Purposes, so you can find a lot of alternatives there, take a look at Tatum, I think they're Great, Good luck!
